tests/fr/_fixtures.yaml

```
@@ -1,22 +1,28 @@
 language: fr
 areas:
-  - name: Cuisine
+  - name: cuisine
     id: kitchen
-  - name: Salon
+  - name: salon
     id: living_room
-  - name: Chambre
+  - name: chambre
     id: bedroom
-  - name: Garage
+  - name: garage
     id: garage
-  - name: "Entrée"
-    id: "hall"
+  - name: entrée
+    id: hall
 entities:
-  - name: lumière de la chambre
+  - name: lumière du plafond
     id: light.bedroom_lamp
     area: bedroom
+  - name: lampe de chevet
+    id: light.bedside_lamp
+    area: bedroom
   - name: interrupteur de la cuisine
     id: switch.kitchen
     area: kitchen
   - name: ventilateur de plafond
     id: fan.ceiling
     area: living_room
+  - name: thermostat du salon
+    id: temperature.living_room
```

```suggestion
    id: climate.living_room
```
`temperature` is not a domain